File: spark-scala/spark-dataframe/dataframe-column/src/main/scala/com/mahfooz/dataframe/columns/remove/RemovingColumn.scala
package com.mahfooz.dataframe.columns.remove

import org.apache.spark.sql.SparkSession

object RemovingColumn {

  def main(args: Array[String]): Unit = {

    val spark = SparkSession.builder
      .appName(getClass.getSimpleName)
      .master("local")
      .getOrCreate()

    val movies = spark.read.json("D:\\data\\movies\\movies.json")

    movies.drop("movie_title")
      .show(2)

  }
}

File: spark-java/spark-dataframe/dataframe-column/src/main/java/com/mahfooz/df/columns/removal/RemovingColumn.java
package com.mahfooz.df.columns.removal;

import org.apache.spark.sql.Dataset;
import org.apache.spark.sql.Row;
import org.apache.spark.sql.SparkSession;

public class RemovingColumn {
    public static void main(String[] args) {
        SparkSession spark = SparkSession
                .builder()
                .master("local[*]")
                .appName("RenamingColumn")
                .getOrCreate();

        String dataHome = System.getenv("DATA_HOME");

        Dataset<Row> df = spark.read()
                .option("header", "true")
                .option("inferSchema", "true")
                .csv(dataHome+"\\FileData\\Csv\\Retail\\all\\online-retail-dataset.csv");

        Dataset<Row> withoutColumnDF = df.drop("Description");
        withoutColumnDF.show();
    }
}
